Formula/g/g-ls.rb

```diff
@@ -0,0 +1,19 @@
+class GLs < Formula
+  desc "Powerfull cross-platform ls"
+  homepage "https://g.equationzhao.space"
+  url "https://github.com/Equationzhao/g/archive/refs/tags/v0.16.0.tar.gz", tag: "v0.16.0"
```

Comment:
```suggestion
  url "https://github.com/Equationzhao/g/archive/refs/tags/v0.16.0.tar.gz"
```

This URL points to a tarball, but `tag` implies that it is a Git repository. Hence the audit failure you observed.